PythonDZ1/DZ1.4.py

```diff
@@ -0,0 +1,18 @@
+#Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, 
+#если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+
+#*Пример:*
+
+#3 2 4 -> yes
+#3 2 1 -> no
+
+n=int(input('Введите число долек в длину:'))
+m=int(input('Введите число долек в ширину:'))
+k=int(input('Введите число сколько долек надо отломить:'))
+
+if k/m%2==0 or k/n%2==0 or k==m or k==n:
+    print('yes')
+else:
+    print('no')
```

Comment:
```suggestion
n = int(input('Введите число долек в длину:'))
m = int(input('Введите число долек в ширину:'))
k = int(input('Введите число сколько долек надо отломить:'))

if k / m % 2 == 0 or k / n % 2 == 0 or k == m or k == n:
    print('yes')
else:
    print('no')
```
Разве мы сможем осуществить надлом в этом случае?
![image](https://user-images.githubusercontent.com/49741194/223320199-43254c46-8f2f-4a03-94df-be1f38c74ec8.png)